algorithms/lithuanian/stem_ISO_8859_1.sbl

```diff
@@ -0,0 +1,380 @@
+externals ( stem )
+/* Special characters in Unicode Latin-1 and Latin Extended-A */
+// ' nosine
+stringdef a'   decimal '261'  // ą a + ogonek
+stringdef e'   decimal '281'  // ę e + ogonek
+stringdef i'   decimal '303'  // į i + ogonek
+stringdef u'   decimal '371'  // ų u + ogonek
+
+// . taskas
+stringdef e.   decimal '279'  // ė e + dot
+
+// - ilgoji
+stringdef u-   decimal '363'  // ū u + macron
+
+// * varnele
+stringdef c*   decimal '269'  // č c + caron (haček)
+stringdef s*   decimal '353'  // š s + caron (haček)
+stringdef z*   decimal '382'  // ž z + caron (haček)
+
+// [C](VC)^m[V|C]
+// definitions of variables for
+// p1 - position of m = 0
+// p2 - position of m = 1
+integers ( p1 p2 s)
+
+// booleans - to be commented
+// CHANGE
+booleans ( CHANGE )
```

Comment:
Although `CHANGE` is set in several places, its value is never actually used anywhere.

The snowball compiler ought to be able to detect such cases and issue a warning, but it doesn't currently.